MVPproject/MVPproject/RecipesModule/Cell/TypeRecipesCollectionViewCell.swift

```diff
@@ -0,0 +1,98 @@
+// TypeRecipesCollectionViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Ячека типа рецепта
+class TypeRecipesCollectionViewCell: UICollectionViewCell {
+    // MARK: - Constants
+
+    static let reuseID = "TypeRecipesCollectionViewCell"
+
+    // MARK: - Visual Components
+
+    private let titleImageView = UIImageView()
+
+    private let bottomView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .darkGray
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.alpha = 0.7
+        return view
+    }()
+
+    private let bottomLabel: UILabel = {
+        let label = UILabel()
+        label.font = .verdana(ofSize: 20)
+        label.adjustsFontSizeToFitWidth = true
+        label.textColor = .white
+        label.textAlignment = .center
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    // MARK: - Initializators
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        backgroundColor = .red
+        setViews()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+
+    // MARK: - Public methods
+
+    func setupCell(type: RecipesType) {
```

Comment:
```suggestion
    func setupCell(category: RecipesCategory) {
```
P.S. Если тебе кажется, что категория неудачное название, но все связанные с этим комменты можно скипать